XMonad/Actions/CopyWindow.hs

```diff
@@ -79,6 +79,8 @@ import qualified XMonad.StackSet as W
 -- To distinguish workspaces containing copies of the focused window use
 -- something like:
 --
+-- Case 1: Using 'spawnPipe'
```

Comment:
I say we drop all the `spawnPipe` stuff. We don't want people using that

Reply:
Thanks for your suggestion. I have deleted that.